e2e/validation.go

```diff
@@ -110,7 +110,7 @@ func commonLiveVMValidators(opts *scenarioRunOpts) []*LiveVMValidator {
 				"cluster-provision-cse-output.log",
 				"cloud-init-files.paved",
 				"vhd-install.complete",
-				"cloud-config.txt",
+				//"cloud-config.txt", // file with UserData
```

Comment:
left over TODO ? no sure why this needs to be removed

Reply:
This file doesn't exist for scriptless approach. Because we don't supply UserData at all. (It mostly contains bash script).